README.md

```diff
@@ -15,12 +15,13 @@
 - To start the electron app: `npm run electron`
 
 ## Package
-- To build a package:`npm install && npm run ng-high-memory` and then: `npm run dist`
+- To build a package:`npm install --save-dev && npm run ng-high-memory` and then: `npm run dist`
 - The packages will be placed in `../output`
-- Example: `npm install && npm run build-prod && npm run dist` will make a production-build installer for Windows
+- Example: `npm install --save-dev && npm run build-prod && npm run dist` will make a production-build installer for Windows
```

Comment:
We don't want `--save-dev` in these instructions. That's for installing new packages to a project and saving them to the dependencies in package.json